openapi/anchor-platform/Callbacks API.yml

```diff
@@ -466,3 +466,34 @@ paths:
             application/json:
               schema:
                 $ref: './schemas.yml#/components/schemas/Error'
+  /events:
+    post:
+      tags:
+        - Events
+        - SEP-24
+        - SEP-31
+      summary: Receive a event from the anchor platform.
+      operationId: postEvent
+      description: |
+        Receive a JSON object representing an event.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                timestamp:
+                    type: string
+                    description: |
+                      `timestamp` the current Unix timestamp (number of seconds since epoch) at the time the callback is sent.
+                payload:
+                  $ref: './schemas.yml#/components/schemas/Event'
```

Comment:
Is now a good time to revisit the schema?

- Is the `id` attribute specific to the event or transaction?
- Why do we have `sep` as a top level attribute when it is available inside the `transaction` object?
- Why do we have a `transaction_error` type? Can't we merge this with `transaction_status_changed`?

Reply:
Also, do we need a timestamp on each event to ensure business can check if they've already processed an event after the one received? Is that possible?

Reply:
The `Event` schema can be further improved. I think we have a couple of days to review. 

For `id`, `sep`, and the transaction status, I am open to redesign if that's also ok with you.

And we should have a `created_at` in the `Event` schema.

Reply:
Sounds good to me.

Reply:
If we redesigning event schema in the separate PR this change LGTM